packages/appkit/src/client.ts

```diff
@@ -1328,6 +1334,14 @@ export class AppKit {
     adapter.on('disconnect', this.disconnect.bind(this))
 
     adapter.on('pendingTransactions', () => {
+      const totalCalls = this.totalCalls.pendingTransactions
```

Comment:
does this need to be reset somewhere? (I know it's a draft PR)

Reply:
+1 if user disconnects we should reset I think. Else this is naturally reset on adapter instantiation (page refresh)